win/stdatomic.c

```diff
@@ -3,6 +3,11 @@
 
 #include "stdatomic.h"
 
+uint_least64_t atomic_exchange(atomic_uint_least64_t *object, uint_least64_t desired)
```

Comment:
i think you should do something like:
`#define atomic_exchange(ptr, desired) (atomic_exchange## sizeof((typeof(ptr)*)ptr)(ptr, desired))`
so this define should translate atomic_exchange into one of

1. atomic_exchange16 
2. atomic_exchange32 
3. atomic_exchange64

and you can implement each one with the specific _InterlockedExchange16 _InterlockedExchange _InterlockedExchange64